delfin-kubernetes/delfin-alert-deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
```

Comment:
Please add a readme in the example folder to describe what is this delfin=kubernetes. Also please provide the link to the documentation where we add the guide.

Reply:
Added readme.MD

Reply:
added readme file